conf/modules.config

```diff
@@ -263,13 +263,13 @@ process {
     }
 
     withName: METASPADES {
-        ext.args   = params.spades_options ?: '--meta'
+        ext.args   = params.spades_options ? params.spades_options + ' --meta' : '--meta'
         ext.prefix = { "SPAdes-${meta.id}" }
         publishDir = [path: { "${params.outdir}/Assembly/SPAdes" }, mode: params.publish_dir_mode, pattern: "*.{fasta.gz,gfa.gz,fa.gz,log}"]
     }
 
     withName: METASPADESHYBRID {
-        ext.args   = params.spades_options ?: '--meta'
+        ext.args   = params.spades_options ? params.spades_options + '--meta' : '--meta'
```

Comment:
```suggestion
        ext.args   = params.spades_options ? params.spades_options + ' --meta' : '--meta'
```

> also add below.

;)

Reply:
Oh that's what you meant I thought it was misplaaced suggestion 🤦🏼‍♀️